Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,118 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.x86_64-darwin9.tar.gz"
```

Comment:
The default on 10.5 is 32-bit not 64-bit and both 10.5 and 10.6 do not mandate a 64-bit processor.